torch_xla/csrc/aten_cpu_fallback.cpp

```diff
@@ -0,0 +1,48 @@
+#include "torch_xla/csrc/aten_cpu_fallback.h"
+
+#include <tensorflow/compiler/xla/xla_client/debug_macros.h>
+#include <tensorflow/compiler/xla/xla_client/metrics.h>
+#include <tensorflow/compiler/xla/xla_client/tf_logging.h>
+#include <torch_xla/csrc/function_call_tracker.h>
+
+#include <unordered_map>
+
+namespace torch_xla {
+
+std::unordered_map<std::string, ::xla::metrics::Counter*>
```

Comment:
I think we should make this map static since we don't expect code outside of this file to access it. wdyt?

Reply:
good catch, static sounds good